packages/support/lib/logging.js

```diff
@@ -35,26 +35,31 @@ export function getLogger(prefix = null) {
     unwrap: () => logger,
     levels: LEVELS,
     prefix,
-    /**
-     * Logs given arguments at the error level and returns
-     * the error object.
-     *
-     * @param  {...any} args
-     * @returns {Error}
-     */
     errorWithException (/** @type {any[]} */ ...args) {
       this.error(...args);
       // make sure we have an `Error` object. Wrap if necessary
       return _.isError(args[0]) ? args[0] : new Error(args.join('\n'));
     },
-    /**
-     * @deprecated Use {@link errorWithException} instead
-     * @param {...any} args
-     * @throws {Error}
-     */
     errorAndThrow (/** @type {any[]} */ ...args) {
       throw this.errorWithException(args);
     },
+    updateCurrentContext(
```

Comment:
I think it still makes sense to have this method implemented inside `@appium/logger`. We may simply have a wrapper for it here

Reply:
I moved this implementation to `@appium/logger`.